packages/web-components/fast-router/ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,3 @@
+# Acknowledgements
+
+A huge thank-you to Ember for the original implementation of the route recognizer so many years ago. That implementation was evolved by Aurelia 1 for its purposes and then later by Aurelia 2 for its purposes. FAST has continued that progression by leveraging the improvements made along the way and then customizing them for its own needs, with much appreciation for the innovation and hard work of these mentioned forerunners.
```

Comment:
Linking to Ember in a new window may help our indexing rank for SEO as it helps to establish credibility.

Reply:
Adding some links now.